### PR TITLE
Added job type to prismic and partner type

### PR DESCRIPTION
**Pull request by AndrewBestbier**
### Pre-flight checklist

- [ ] Unit tests
- [ ] GraphiQL tested
- [ ] Client app tested
- [ ] Gif added

### CODE REVIEW

**Comment by pixelblend** [comment]
Dependent on https://github.com/redbadger/badger-brain/pull/67

File: lib/fetch/sanitize.js [hunk 1]
```diff
@@ -80,6 +80,7 @@ export function sanitizePartner(json) {
     description: get('partner.description').substr(0, 50),
     partnerURL: get('partner.partnerURL'),
     imageURL: get('partner.imageURL'),
+    jobs: get('partner.jobs'),
```
Comment (lpil) [inline lib/fetch/sanitize.js]: Does this need to default to empty array instead of null?

File: lib/partner/partnerFields.js [hunk 2]
```diff
@@ -28,6 +31,11 @@ const PartnerTypeFields = {
     type: new GraphQLNonNull(GraphQLString),
     description: 'URL for partner logo',
   },
+  jobs: {
+    type: new GraphQLList(JobType),
+    description: 'List of partner jobs',
```
Comment (lpil) [inline lib/partner/partnerFields.js]: As in jobs one could apply for at the partner company? Are these job adverts?

Comment (AndrewBestbier) [inline lib/partner/partnerFields.js]: Yep, sorry should have made more clear in the description. This is in relation to [this](https://trello.com/c/EaiIO5ck/28-sponsors-page-react-london-conf) ticket on trello

File: lib/fetch/sanitize.js [hunk 3]
```diff
@@ -77,9 +77,10 @@ export function sanitizePartner(json) {
     id: json.uid,
     name: get('partner.name'),
     level: get('partner.level'),
-    description: get('partner.description').substr(0, 50),
+    description: get('partner.description'),
```
Comment (AndrewBestbier) [inline lib/fetch/sanitize.js]: After a discussion with Alex it was decided that it is up to the client to limit the description, not badger brain.

Comment (lpil) [inline lib/fetch/sanitize.js]: Sounds good. :)

**Comment by lpil** [comment]
Looks great :D

**Pull request merged**